### PR TITLE
Hylian (Twilight Princess)

### PR DESCRIPTION
**Pull request by Mike-3**

### CODE REVIEW

File: lib/widgets/registry.dart [hunk 1]
```diff
@@ -1416,6 +1416,12 @@ class Registry {
         iconPath: SYMBOLTABLES_ASSETPATH + 'color_tokki/72.png',
         searchStrings: SEARCHSTRING_SYMBOLTABLES + 'colortokki color-tokki farbcode woven carpet webteppich gewebter farben colors six '
       ),
+      GCWToolWidget(
+          tool: SymbolTable(symbolKey: 'cyrillic'),
```
Comment (S-Man42) [inline lib/widgets/registry.dart:1420]: Cyrillic seems to be case sensitive. The default case is, for a and A the same symbol is shown. But if you have case sensitive font, the symbol table widget needs to know about it to make a difference. For that, there is a field "caseSensitve" for the SymbolTable constructor. (have a look at romulan or klingon). Does this fix the problem you mentioned?

File: assets/i18n/de.json [hunk 2]
```diff
@@ -1718,6 +1718,51 @@
   "symboltables_color_honey_description" : "Sechs-Farben-Code, Bienenwaben",
   "symboltables_color_tokki_title" : "Color-Tokki",
   "symboltables_color_tokki_description" : "Sechs-Farben-Code, Webteppich",
+  "symboltables_cyrillic_title" : "Kyrillisches Alphabet",
+  "symboltables_cyrillic_description" : "russische Schrift",
```
Comment (S-Man42) [inline assets/i18n/de.json:1722]: Please start the description with a capital letter :)

**Review by S-Man42 — COMMENTED** [review]
Just some minor mentions. Looks great anyway!

File: assets/i18n/de.json [hunk 3]
```diff
@@ -1893,6 +1938,10 @@
   "symboltables_hvd_title" : "HVD",
   "symboltables_hylian_skyward_sword_title" : "Hylianisch (Skyward Sword)",
   "symboltables_hylian_skyward_sword_description" : "Aus dem Videospiel 'The Legend of Zelda: Skyward Sword'",
+  "symboltables_hylian_twilight_princes_gcn_title" : "Hylianisch (Twilight Princess)",
```
[the remark this answers is not in the document]
Comment (Mike-3) [reply assets/i18n/de.json:1942]: changed to Nintendo GameCube

File: lib/widgets/registry.dart [hunk 4]
```diff
@@ -1608,6 +1614,18 @@ class Registry {
         iconPath: SYMBOLTABLES_ASSETPATH + 'hylian_skyward_sword/79.png',
         searchStrings: SEARCHSTRING_SYMBOLTABLES + 'hylianische skywardsword skywardschwert thelegendofzelda hyrule '
       ),
+      GCWToolWidget(
+        tool: SymbolTable(symbolKey: 'hylian_twilight_princes_gcn'),
+        i18nPrefix: 'symboltables_hylian_twilight_princes_gcn',
```
[the remark this answers is not in the document]
Comment (Mike-3) [reply lib/widgets/registry.dart:1625]: changed

File: lib/widgets/registry.dart [hunk 5]
```diff
@@ -1416,6 +1416,12 @@ class Registry {
         iconPath: SYMBOLTABLES_ASSETPATH + 'color_tokki/72.png',
         searchStrings: SEARCHSTRING_SYMBOLTABLES + 'colortokki color-tokki farbcode woven carpet webteppich gewebter farben colors six '
       ),
+      GCWToolWidget(
+          tool: SymbolTable(symbolKey: 'cyrillic'),
+          i18nPrefix: 'symboltables_cyrillic',
+          iconPath: SYMBOLTABLES_ASSETPATH + 'cyrillic/68.png',
+          searchStrings: SEARCHSTRING_SYMBOLTABLES + 'kyrillisch cyrillic russisch russian alphabet schrift font cyrl kyrill saloniki'
```
[the remark this answers is not in the document]
Comment (Mike-3) [reply lib/widgets/registry.dart:1423]: changed
kyrillisches ?

**Pull request closed without merging**